<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google API Latitude Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/images/ics.gif</normaloff>:/images/images/ics.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHistory_Locations"/>
    <addaction name="actionShow_Current_Location"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShow_Current_Location">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/location-icon.png</normaloff>:/images/images/location-icon.png</iconset>
   </property>
   <property name="text">
    <string>Show Current Location</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHistory_Locations">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/history-locations.png</normaloff>:/images/images/history-locations.png</iconset>
   </property>
   <property name="text">
    <string>History Locations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
